frontend/src/components/classify-failures.js

```diff
@@ -156,27 +119,20 @@ export class ClassifyFailuresTable extends React.Component {
   }
 
   setFilter = (field, value) => {
-    this.updateFilters(field, 'eq', value, () => {
-      this.refreshResults();
-    })
+    // maybe process values array to string format here instead of expecting caller to do it?
+    let operator = (value.includes(";")) ? 'in' : 'eq'
+    this.updateFilters(field, operator, value, this.refreshResults)
   };
 
   removeFilter = id => {
-    if (id === "metadata.exception_name") {   // only remove exception_name filter
-      this.updateFilters(id, null, null, () => {
-        this.setState({exceptionSelections: [], page: 1}, this.applyExceptionFilter);
-      });
+    if ((id !== "result") && (id !== "run_id")) {   // Don't allow removal of error/failure filter
+      this.updateFilters(id, null, null, this.refreshResults)
     }
-  }
+  };
```

Comment:
I know we're inconsistent here, but the semi-colon is technically optional. Don't worry about it in the future :smile: